.storybook/preview.tsx

```diff
@@ -6,13 +6,28 @@ import { getSdkComponentMap } from '@pega/react-sdk-components/lib/bridge/helper
 import { theme } from '../src/theme';
 
 import { decorator } from '../__mocks__/react_pconnect';
+import setPCoreMocks from '../__mocks__/pcoreMocks';
+
+declare global {
+  interface Window {
+    PCore?: any;
+  }
+}
+
+declare global {
+  interface Window {
+    PCore?: any;
+  }
+}
```

Comment:
looks like global declaration is duplicated

Reply:
Deleted.